client/src/features/ProjectPageV2/ProjectPageNav/ProjectPageNav.module.scss

```
@@ -0,0 +1,35 @@
+@import "~bootstrap/scss/functions";
+@import "~bootstrap/scss/variables";
+@import "~bootstrap/scss/variables-dark";
+@import "~bootstrap/scss/maps";
+@import "~bootstrap/scss/mixins";
+
+.ProjectPageNavItemActive a {
+  font-weight: bold;
+  text-decoration: underline;
+  text-underline-offset: 5px;
+  padding: 0;
+  margin-bottom: 0;
+  color: var(--bs-rk-text);
+}
+
+.ProjectPageNavItem a {
+  color: var(--bs-rk-text);
+}
+.ProjectPageNavItem a:hover {
+  text-decoration: underline;
+  text-underline-offset: 5px;
+}
+
+@include media-breakpoint-up(lg) {
+  .ProjectPageNavItem a {
+    color: var(--bs-rk-text-light);
+  }
+  .ProjectPageNavItem a:hover {
+    color: var(--bs-rk-text);
+  }
+
+  .ProjectPageNavItemActive a {
+    color: var(--bs-rk-text);
+  }
+}
```

The styles are applied at the wrong level here, they need to be applied to the `a.nav-link` component.

```suggestion
.navLink {
  color: var(--bs-rk-text);

  &:hover {
    text-decoration: underline;
    text-underline-offset: 5px;
  }

  &:global(.active) {
    font-weight: bold;
    text-decoration: underline;
    text-underline-offset: 5px;
    padding: 0;
    margin-bottom: 0;
    color: var(--bs-rk-text);
  }

  @include media-breakpoint-up(lg) {
    & {
      color: var(--bs-rk-text-light);

      &:hover {
        color: var(--bs-rk-text);
      }
    }

    &:global(.active) {
      color: var(--bs-rk-text);
    }
  }
}
```